packages/node_modules/@node-red/runtime/lib/flows/Flow.js

```diff
@@ -404,6 +416,144 @@ class Flow {
         return this.activeNodes;
     }
 
+    /**
+     * Get a flow setting value deined in this flow. 
+     * @param  {String} key
+     * @return {Object} result containinig val property or null
+     */
+    getFlowSetting(name) {
```

Comment:
We already have `getSetting` defined on the `Flow` object. Why have you separated this out? I can see `getSetting` now calls this function - but I cannot see it being called from anywhere else.

Reply:
Merged them, as there is no particular reason for them to be separated.